From 9e4d46611c4d4600a5abac81b619ae1489d4b8c1 Mon Sep 17 00:00:00 2001
From: Jacob Stiffler <j-stiffler@ti.com>
Date: Mon, 27 Apr 2015 08:27:59 -0400
Subject: [PATCH 6/6] GstDRMBufferPool support fix:

Conditionally build drm library based on if kmssink dependencies are
by pkgconfig found.

Signed-off-by: Jacob Stiffler <j-stiffler@ti.com>
---
 gst-libs/gst/Makefile.am |    8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/gst-libs/gst/Makefile.am b/gst-libs/gst/Makefile.am
index b915d07..52ff331 100644
--- a/gst-libs/gst/Makefile.am
+++ b/gst-libs/gst/Makefile.am
@@ -2,9 +2,13 @@ if HAVE_EGL
 EGL_DIR = egl
 endif
 
-SUBDIRS = interfaces basecamerabinsrc codecparsers drm \
+if USE_KMS
+DRM_DIR = drm
+endif
+
+SUBDIRS = interfaces basecamerabinsrc codecparsers $(DRM_DIR) \
 	 insertbin uridownloader mpegts $(EGL_DIR)
 
 noinst_HEADERS = gst-i18n-plugin.h gettext.h glib-compat-private.h
-DIST_SUBDIRS = interfaces egl basecamerabinsrc codecparsers \
+DIST_SUBDIRS = interfaces egl basecamerabinsrc codecparsers drm \
 	insertbin uridownloader mpegts
-- 
1.7.9.5

